src/sparseml/transformers/finetune/session_mixin.py

```diff
@@ -107,14 +105,17 @@ def __init__(
         self.criterion = torch.nn.CrossEntropyLoss()
 
         model_signature = inspect.signature(self.model.forward)
-        self._model_signature_columns = list(model_signature.parameters.keys())
+        self._signature_columns = list(model_signature.parameters.keys())
```

Comment:
is this due to transformers version?

Reply:
Yep, not our recent upgrade though. I had kept this name in from the old trainer code but realized it got renamed a few versions ago